src/dlgTriggerEditor.cpp

```diff
@@ -193,6 +193,50 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 
     mudlet::loadEdbeeTheme(mpHost->mEditorTheme, mpHost->mEditorThemeFile);
 
+    edbee::StringTextAutoCompleteProvider* provider = new edbee::StringTextAutoCompleteProvider();
```

Comment:
Change this to a `QScopedPointer` so we don't have to worry about a memory leak when the profile is closed